tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -224,7 +226,8 @@ spec:
         - "$(params.WORKERS)"
         # NOTE: The syntax below is required to negate boolean parameters
         - "--info=$(params.INFO)"
-        - "--timeout=$(params.TIMEOUT)"
+        # No timeout for EC, though Tekton may apply a timeout
+        - "--timeout=0"
```

Comment:
This works quite badly on any version of EC that doesn't have this PR merged. It's probably fine  (???), but I was considering using `--timeout=100h` for a while just to be extra cautious/paranoid.

Reply:
We can't release this version of the task without the new version of EC